### ISSUE TITLE
Research feedback from previous SiS clients

### ISSUE THREAD

**Issue by joeniquette** [department-of-veterans-affairs/va.gov-team#74894]
### Issue Description

review feedback from previous SiS integrations - synthesize for recommendations (github docs, sensitive repo)

**Comment by parkerbees** [comment]
@joeniquette is this related to the main SiS product? Trying to determine relevance/priority. Thx.

**Comment by joshmatulonis** [comment]
I was assigned to collecting feedback but I focused on VAmobile slack channel. I think the FAQ ticket included collect feedback, or I added the feedback task to it in case I came across feedback while reviewing the channel's threads. Something happened where the task became split up between me and another. It may have been that feedback wasn't found in GH. But the ticket is assigned to me and I'll be happy to complete/provide update on it immediately.
